### PR TITLE
Added agent uptime to status log and status gui

### PR DESCRIPTION
**Pull request by albertvaka**
### What does this PR do?

Adds the agent uptime to the status log and status gui

### Motivation

While using a flare, if the log files have already rotated, you can't easily get the Agent uptime, which could be helpful to debug a problem.

### CODE REVIEW

File: pkg/status/status.go [hunk 1]
```diff
@@ -26,8 +27,21 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/version"
 )
 
+var startTime = time.Now()
 var timeFormat = "2006-01-02 15:04:05.000000 MST"
 
+// FormatUptime formats a duration into a human-readable, uptime-like string
```
Comment (remeh) [inline pkg/status/status.go]: As the start time will be used to diagnose problems, I'm wondering if having a more raw display would be more efficient (ie. 2006-01-02 15:04:05.000000 MST), what do you think?

**Review by remeh — DISMISSED** [review]
Left a comment, looks great otherwise!

File: pkg/status/status.go (hunk 1 above)
Comment (albertvaka) [reply pkg/status/status.go]: Removed the custom formatting

**Review by remeh — APPROVED** [review]
:shipit:

**Pull request merged**